### PR TITLE
OCM-11556 | fix: refactor multiple ca tests from getClusterRegistryConfig

### PR DESCRIPTION
**Pull request by chenz4027**
https://issues.redhat.com/browse/OCM-11556

### CODE REVIEW

**Review by andreadecorte — APPROVED** [review]

File: cmd/describe/cluster/cmd_test.go [hunk 1]
```diff
@@ -149,6 +151,12 @@ var _ = Describe("getClusterRegistryConfig", func() {
 				DomainName("quay.io").Insecure(true)).
 			PlatformAllowlist(cmv1.NewRegistryAllowlist().ID("test-id"))).Build()
 		Expect(err).NotTo(HaveOccurred())
+		mockCaOutput := ""
+		for registry := range mockCa {
+			mockCaOutput = fmt.Sprintf("%s"+
+				"    - %s: REDACTED\n", mockCaOutput, registry)
+
+		}
```
Comment (gdbranco) [inline cmd/describe/cluster/cmd_test.go:159]: hmm, I meant to use sprintf directly on the map, but you can't do it because of redacted is that it?
I suggest to pull the map keys, sort it then use that list instead of the map to go through the keys and write the redacted alongside it then :thinking: 
Getting the keys via a loop on the map directly results in it being able to be out of order again

Comment (chenz4027) [reply cmd/describe/cluster/cmd_test.go:159]: I see, thanks @gdbranco , let me try again

Comment (chenz4027) [reply cmd/describe/cluster/cmd_test.go]: Updated, @gdbranco mind taking another look?
